### PR TITLE
First Update

### PR DESCRIPTION
**Pull request by braedenmc1**
I added my comments to the survey last week!

### CODE REVIEW

File: survey.Rmd [hunk 1]
```diff
@@ -1,20 +1,71 @@
 ---
 title: "survey"
-author: "Rubén A. Rodríguez Barrón"
+author: "Braeden McNulty (Florida State University) & Rubén A. Rodríguez Barrón (University of Chicago)"
```
Comment (rrodriguezbarron) [inline survey.Rmd:3]: Yeah, this is ok.

File: survey.Rmd [hunk 2]
```diff
@@ -1,20 +1,71 @@
...
 output:
   html_document:
     toc: true
 ---
+
+# Geographic Measures
+
+1. RUCA - ZIP Code [INTEGER - ORDINAL]
+These measure the location of ZIP codes along the urban-rural continuum. They account for how interconnected ZIP codes are to urbanized areas through roadways, commuter paths, and other infrastructure.
+    - 1 (Most Urban)
```
Comment (rrodriguezbarron) [inline survey.Rmd:14]: Braeden, do we need 10? Could it be just 5?

Comment (braedenmc1) [reply survey.Rmd:14]: We can and should collapse these, this is just how it would look in the raw file.

Comment (rrodriguezbarron) [reply survey.Rmd:14]: Agreed!

**Pull request merged**